src/kayenta/metricStore/datadog/metricConfigurer.tsx

```diff
@@ -43,7 +79,8 @@ function mapStateToProps(state: ICanaryState): IDatadogMetricConfigurerStateProp
 
 function mapDispatchToProps(dispatch: (action: Action & any) => void): IDatadogMetricConfigurerDispatchProps {
   return {
-    changeMetricName: (metricName: string): void => {
+    changeMetricName: (agg: string, name: string): void => {
+      var metricName = agg + ':' + name;
```

Comment:
Sorry one small nit -- could you use `const` here instead of `var`?

Reply:
Tweaked.

Reply:
Thank you!